src/30_Advanced/ComboBox.html

```diff
@@ -12,32 +12,29 @@
 <head>
   <title>ComboBox</title>
   <meta charset="utf-8">
-  <link rel="canonical" href="<%host%>/..\30_advanced/combobox/">
+  <link rel="canonical" href="https://ampbyexample.com/..\30_advanced/combobox/">
```

Comment:
why do we need to specify the production URL?

Reply:
good find. done